test/CMakeLists.txt

```diff
@@ -1,226 +1,221 @@
 target_link_libraries(TransitivePathTest engine gtest_main ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(TransitivePathTest)
 
 add_executable(SparqlLexerTest SparqlLexerTest.cpp)
-add_test(SparqlLexerTest SparqlLexerTest)
 target_link_libraries(SparqlLexerTest parser gtest_main ${CMAKE_THREAD_LIBS_INIT} ${ICU_LIBRARIES})
+gtest_discover_tests(SparqlLexerTest)
 
 set(TEST_COMPILE_TIME_REGEXES OFF CACHE BOOL "Include the tests for the Utf8Regex.")
 if (TEST_COMPILE_TIME_REGEXES)
   add_executable(Utf8RegexTest Utf8RegexTest.cpp)
-  add_test(Utf8RegexTest Utf8RegexTest)
   target_link_libraries(Utf8RegexTest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+  gtest_discover_tests(Utf8RegexTest)
 endif (TEST_COMPILE_TIME_REGEXES)
 
 add_executable(BatchedPipelineTest BatchedPipelineTest.cpp)
-add_test(BatchedPipelineTest BatchedPipelineTest)
 target_link_libraries(BatchedPipelineTest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(BatchedPipelineTest)
 
 add_executable(TupleHelpersTest TupleHelpersTest.cpp)
-add_test(TupleHelpersTest TupleHelpersTest)
 target_link_libraries(TupleHelpersTest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(TupleHelpersTest)
 
 add_executable(StringSortComparatorTest StringSortComparatorTest.cpp)
-add_test(StringSortComparatorTest StringSortComparatorTest)
 target_link_libraries(StringSortComparatorTest gtest_main ${CMAKE_THREAD_LIBS_INIT} ${ICU_LIBRARIES})
+gtest_discover_tests(StringSortComparatorTest)
 
 add_executable(PriorityQueueTest PriorityQueueTest.cpp)
-add_test(PriorityQueueTest PriorityQueueTest)
 target_link_libraries(PriorityQueueTest gtest_main absl::flat_hash_map ${CMAKE_THREAD_LIBS_INIT} ${ICU_LIBRARIES})
+gtest_discover_tests(PriorityQueueTest)
 
 add_executable(SynchronizedTest SynchronizedTest.cpp)
-add_test(SynchronizedTest SynchronizedTest)
 target_link_libraries(SynchronizedTest gtest_main ${CMAKE_THREAD_LIBS_INIT})
-
-add_executable(BindTest BindTest.cpp)
-add_test(BindTest BindTest)
-target_link_libraries(BindTest gtest_main engine ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(SynchronizedTest)
 
 add_executable(LimitedAllocatorTest LimitedAllocatorTest.cpp)
-add_test(LimitedAllocatorTest LimitedAllocatorTest)
 target_link_libraries(LimitedAllocatorTest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(LimitedAllocatorTest)
 
 add_executable(MinusTest MinusTest.cpp)
-add_test(MinusTest MinusTest)
 target_link_libraries(MinusTest gtest_main engine ${CMAKE_THREAD_LIBS_INIT})
+gtest_discover_tests(MinusTest)
 
 add_executable(SortPerformanceEstimatorTest SortPerformanceEstimatorTest.cpp)
 target_link_libraries(SortPerformanceEstimatorTest gtest_main SortPerformanceEstimator ${CMAKE_THREAD_LIBS_INIT})
 # this test runs for quite some time and might have spurious failures! If this is undesired, comment it out
-#add_test(SortPerformanceEstimatorTest SortPerformanceEstimatorTest)
+#gtest_discover_tests(SortPerformanceEstimatorTest)
```

Comment:
@joka921 This test is currently unused. Should the status-quo be kept, or the test changed so it actually works as a unit-test?

Reply:
Just leave this as is.